src/app/sessions/states/sign-in/sign-in.coffee

```diff
@@ -27,6 +29,9 @@ angular.module("doubtfire.sessions.states.sign-in", [])
 
   $scope.session = { remember_me: true }
 
+  # Get the confugurable, external name of Doubtfire
+  $scope.externalName = configurationService.getExternalName()
```

Comment:
Here we could just inject `externalName` constant and then expose to our `$scope`:

```
.controller("SignInCtrl", ($scope, $state, ..., externalName) ->
  ...
  $scope.externalName = externalName
)
```